scripts/gui/microbe_hud.mjs

```diff
@@ -293,6 +328,9 @@ function hideTipMsg() {
 function onMenuClicked(){
 
     common.playButtonPressSound();
+    document.getElementById("mainMenuButton").classList.add("MainMenuActive");
+    document.getElementById("mainMenuButton").classList.remove("MainMenuNormal");
+    "url('../../Textures/gui/bevel/MainMenuActive.png')";
```

Comment:
This string here seems to do nothing?

Reply:
when menu button si pressed, the menu is active, when resume is clicked menu get to normal

Reply:
Look at line 333. It is just a string (`"";`) and a semicolon. It also doesn't continue from the previous line (line 332 ends with a semicolon). Hence I think this line does absolutely nothing.